<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="IN4-C138" failures="1" tests="1" name="TEST" time="3.772" errors="0" timestamp="2023-01-31T18:21:37 IST">
  <testcase classname="com.RE.Engineering.Test.Issue_List_Test" name="IssueInventry" time="3.756">
    <failure type="java.lang.ExceptionInInitializerError">
      <![CDATA[java.lang.ExceptionInInitializerError
at com.RE.Engineering.Test.Issue_List_Test.IssueInventry(Issue_List_Test.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.NullPointerException: Cannot invoke "org.apache.poi.xssf.usermodel.XSSFSheet.getRow(int)" because "com.RE.Engineering.ObjectRepository.Issue_List.sheet" is null
at com.RE.Engineering.ObjectRepository.Issue_List.<clinit>(Issue_List.java:24)
... 29 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- IssueInventry -->
</testsuite> <!-- TEST -->
